Menus/LoadingScreen.cpp

```diff
@@ -46,6 +46,7 @@ namespace RTE {
 			CreateLoadingSplash();
 		}
 
+		g_WindowMan.ClearRenderer();
```

Comment:
Just gonna point ou the loading screen is a black screen for me on initial load. Dunno if something needs to be inited and the loading happens before then?